### DISCUSSION TITLE
code . in mac terminal produces error when trying to open a directory

### DISCUSSION THREAD

**Discussion by BrandonMoon01** [Homebrew/discussions#3277] [Everyday usage]
I am currently on a MacBook Pro with the M1 chip running macOS Monterey 12.3.1. I am having an issue trying to open directories within my terminal with the "code ." command. Specifically, I am not seeing any of the files that are contained within my directory and I am also getting an error: `shell-init: error retrieving current directory: getcwd: cannot access parent directories: Operation not permitted
job-working-directory: error retrieving current directory: getcwd: cannot access parent directories: Operation not permitted`
I have been dealing with this issue a while now and all the solutions posted online don't seem to be working for me. Any insight is much appreciated! 
<img width="1511" alt="Screen Shot 2022-05-16 at 6 38 06 PM" src="https://user-images.githubusercontent.com/70309037/168699147-404c678f-e81d-4dda-a93c-6f31051eee75.png">

**Comment by SMillerDev** [comment]
Try updating vscode
